user-impl/src/main/java/com/example/auction/user/impl/UserServiceImpl.java

```diff
@@ -68,11 +88,18 @@ public UserServiceImpl(PersistentEntityRegistry registry, ActorSystem system, Ma
                 .thenApply(TreePVector::from);
     }
 
+
     private PersistentEntityRef<UserCommand> entityRef(UUID id) {
         return entityRef(id.toString());
     }
 
     private PersistentEntityRef<UserCommand> entityRef(String id) {
+        registry.register(UserEntity.class);
         return registry.refFor(UserEntity.class, id);
     }
+
+    private PersistentEntityRef<AuthCommand> authRef(String username) {
+        registry.register(AuthEntity.class);
```

Comment:
As a general rule, do all `registry.register(XxxYyZzzzz.class);` on the `MyServiceImpl`  constructor and just once. This is registering `AuthEntity.class` over and over. And same thing applies on line 97.